src/SourceBuild/tarball/content/repos/Directory.Build.targets

```
@@ -376,6 +376,7 @@
           DependsOnTargets="BuildRepoReferences"
           Inputs="$(MSBuildProjectFullPath)"
           Outputs="$(RepoCompletedSemaphorePath)Build.complete">
+    <Exec Command="$(DotnetToolCommand) build-server shutdown" />
```

To me some blank lines would help the readability.

```suggestion

    <Exec Command="$(DotnetToolCommand) build-server shutdown" />

```